<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #f0f8ff;
}

QPushButton {
    background-color: #4CAF50;
    color: white;
    border-radius: 8px;
    padding: 5px;
}

QPushButton:hover {
    background-color: #45a049;
}

QLineEdit {
    border: 2px solid #ccc;
    border-radius: 4px;
    padding: 4px;
    background-color: #ffffff;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame_7">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>141</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>Icons/logo_vit_7_3.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>C R M - ADMIN MENU</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_5">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="admin_menu_buton_activity_control">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #55aaff;
    color: white;
    border-radius: 8px;
    padding: 8px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #8263c6;
}

QPushButton:pressed {
    background-color: #594097;
}
</string>
         </property>
         <property name="text">
          <string>Activity Control</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>Icons/activity_control.png</normaloff>Icons/activity_control.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="admin_menu_buton_sendmail">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #55aaff;
    color: white;
    border-radius: 8px;
    padding: 8px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #8263c6;
}

QPushButton:pressed {
    background-color: #594097;
}
</string>
         </property>
         <property name="text">
          <string>Send Mail</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>Icons/send_mail.png</normaloff>Icons/send_mail.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="admin_menu_buton_prefencemenu">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #55aaff;
    color: white;
    border-radius: 8px;
    padding: 8px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #8263c6;
}

QPushButton:pressed {
    background-color: #594097;
}
</string>
         </property>
         <property name="text">
          <string>Preference Admin Menu</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>Icons/preference_admin.png</normaloff>Icons/preference_admin.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="admin_menu_buton_exit">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #55aaff;
    color: white;
    border-radius: 8px;
    padding: 8px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #8263c6;
}

QPushButton:pressed {
    background-color: #594097;
}
</string>
         </property>
         <property name="text">
          <string>Exit</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>Icons/exit-icon.png</normaloff>Icons/exit-icon.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="admin_menu_table_result">
      <column>
       <property name="text">
        <string>Activity Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Start Time</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Participant Mail</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Organiser Mail</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
